src-tauri/src/tray/mod.rs

```diff
@@ -8,7 +8,7 @@ use tauri::{
         AboutMetadataBuilder, CheckMenuItemBuilder, MenuBuilder, MenuItemBuilder,
         PredefinedMenuItem,
     },
-    tray::{ClickType, TrayIconBuilder},
+    tray::{MouseButton, MouseButtonState, TrayIconBuilder, TrayIconEvent},
```

Comment:
This is for [patch 2.0.0-beta.21](https://tauri.app/releases/) 

The [patch merges this,](https://github.com/tauri-apps/tauri/pull/9777) which introduces the following breaking changes:


>    Changed TrayIconEvent to be an enum instead of a struct.
>     Added MouseButtonState and MouseButton enums.
>     Removed ClickType enum and replaced it with MouseButton enum.
>     Added MouseButtonState enum.